dist_tables/ddl.rst

```diff
@@ -51,7 +51,11 @@ You are now ready to insert data into the distributed table and run queries on i
 Reference Tables
 ~~~~~~~~~~~~~~~~
 
-The above method distributes tables into multiple horizontal shards, but it's also possible to distribute tables into a single shard and replicate it to every worker node. Tables distributed this way are called *reference tables.*  They are typically small non-partitioned tables which we want to locally join with other tables on any worker. One US-centric example is information about states.
+The above method distributes tables into multiple horizontal shards, but another possibility is distributing tables into a single shard and replicating the shard to every worker node.
```

Comment:
Can we start with the motivation here? (e.g. "What about data that is shared amongst tenants?").

I like the opening paragraph here: https://docs.citusdata.com/en/v7.2/use_case_guide/multi_tenant.html#sharing-data-between-tenants.

Also, can we link to the above section from here? (e.g. " See an example app and how we might use reference tables there") (or some such).

Reply:
I was trying to make your suggestion fit with the flow but having trouble. The page as a whole isn't centered around the MT use case, so opening this section talking about sharing data amongst tenants didn't feel right.

However this section does swing around to MT by the second paragraph, and gets into an example in the third.

Good idea about linking to the use case guide. I added the link.